card/mod/05_standard/set/all/comment.rb

```diff
@@ -1,16 +1,22 @@
-event :add_comment, after: :approve, on: :save, when: proc {|c| c.comment } do
+event :add_comment, after: :approve, on: :save, when: proc { |c| c.comment } do
+  cleaned_comment =
+    comment.split(/\n/).map do |line|
+      "<p>#{line.strip.empty? ? '&nbsp;' : line}</p>"
+    end * "\n"
+
+  signature =
+    if Auth.signed_in?
+      "[[#{Auth.current.name}]]"
+    else
+      Env.session[:comment_author] = comment_author if Env.session
+      "#{ comment_author } (Not signed in)"
+    end
+
   self.content = %{
-    #{ content }
-    #{ '<hr>' unless content.blank? }
-    #{ comment.split(/\n/).map {|line| "<p>#{line.strip.empty? ? '&nbsp;' : line}</p>"} * "\n" }
-    <div class="w-comment-author">--#{
-      if Auth.signed_in?
-        "[[#{Auth.current.name}]]"
-      else
-        Env.session[:comment_author] = comment_author if Env.session
-        "#{ comment_author } (Not signed in)"
-      end
-    }.....#{Time.now}</div>
+    #{content}
+    #{'<hr>' unless content.blank?}
+    #{cleaned_comment}
+    <div class="w-comment-author">--#{signature}.....#{Time.now}</div>
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.